services/storage/src/simcore_service_storage/access_layer.py

```diff
@@ -164,6 +181,14 @@ async def get_project_access_rights(
     """
     Returns access-rights of user (user_id) over a project resource (project_id)
     """
+    # when the project is removed, the project will no longer exist
+    # instead of returning AccessRights.none()
+    # an error will be raised
+    # this allows to handle the removal of the project data after the project was
+    # remove from the database
```

Comment:
not sure this comment should be in that location. 
And also, I guess the same could happen with the user id if the user was deleted prior to calling the entrypoint?

Reply:
let me make it more general